### DISCUSSION TITLE
Can I load the card without animation?

### DISCUSSION THREAD

**Discussion by IceTiki** [anuraghazra/github-readme-stats#2172] [Q&A] [answered]
### Is your feature request related to a problem? Please describe.

When I convert markdown file to pdf, the card always not animated ready.

**Comment by rickstaa** [comment, marked as answer]
@IceTiki Thanks for your question. You can use the `disable_animations` parameter. Please check out the [documentation](https://github.com/anuraghazra/github-readme-stats#stats-card-exclusive-options).
